### DISCUSSION TITLE
（Solved）Can you please check my student status？

### DISCUSSION THREAD

**Discussion by liuyangme** [community/community#17811] [GitHub Education]
I submitted my educational email and other documents to apply as you requested but you rejected me, I feel I was treated unfairly, can you please check my student status?

**Comment by liuyangme** [comment]
In the history I submitted, I can see documents including student ID, transcripts, online verification report of school registration, etc. The online verification report of school registration is an official document issued by the Chinese Ministry of Education to prove that I am enrolled in school, and the document also includes a verification code and verification website to prove the authenticity of the document.

**Comment by techvio-dev** [comment]
this just happened for me too

**Reply by liuyangme** [reply]
what should we do

**Comment by suprstarrd** [comment]
https://github.com/github-community/community/discussions/31#discussioncomment-2561585

**Comment by mlarasusan** [comment]
Hello 👋

[👉](https://emojipedia.org/backhand-index-pointing-right/)The support team can help you [here](https://support.github.com/contact/education).
[👉](https://emojipedia.org/backhand-index-pointing-right/)[Here](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/), you can find all the details on how to get a student developer pack 
[👉](https://emojipedia.org/backhand-index-pointing-right/)And here we have a student [review](https://github.com/github-community/community/discussions/31#discussioncomment-2561585) 

[✨](https://emojipedia.org/sparkles/)
